Content.Server/GameObjects/EntitySystems/HandsSystem.cs

```diff
@@ -215,12 +215,9 @@ private bool HandleThrowItem(ICommonSession session, GridCoordinates coords, Ent
 
             physComp.LinearVelocity = dirVec * spd;
 
-            var wHomoDir = Vector3.UnitX;
+            var humanDir = new Angle(dirVec).GetCardinalDir().ToAngle();
 
-            transform.InvWorldMatrix.Transform(ref wHomoDir, out var lHomoDir);
-
-            lHomoDir.Normalize();
-            transform.LocalRotation = new Angle(lHomoDir.Xy);
+            transform.LocalRotation = humanDir;
```

Comment:
at this point why even have the `humanDir` var

Reply:
good point